pom.xml

```diff
@@ -6,15 +6,15 @@
 
     <groupId>se.eris</groupId>
     <artifactId>notnull-instrumenter-maven-plugin</artifactId>
-    <version>1.0.0</version>
+    <version>1.1.0-java-14-support-SNAPSHOT</version>
     <packaging>maven-plugin</packaging>
 
     <description>NotNull annotations instrumenter maven plugin</description>
 
     <url>https://github.com/osundblad/intellij-annotations-instrumenter-maven-plugin</url>
 
     <prerequisites>
-        <maven>3.0.0</maven>
+        <maven>3.1.0</maven>
```

Comment:
The whole `<prerequisites>` element should probably be removed.

From [the Maven POM reference](http://maven.apache.org/pom.html#Prerequisites):
> The POM may have certain prerequisites in order to execute correctly. For example, perhaps there was a fix in Maven 2.0.3 that you need in order to deploy using sftp.
>
> In Maven 3, use Maven Enforcer Plugin's `requireMavenVersion` rule, or other rules to check other aspects.
>
> In Maven 2, here is where you give the prerequisites to building: if these are not met, Maven will fail the build before even starting. The only element that exists as a prerequisite in POM 4.0 is the `maven` element, which takes a minimum version number. **It is checked with Maven 2, it is not any more in Maven 3.**

*(Emphasis on the last sentence added by me)*